examples/interface_examples/procthor.md

```diff
@@ -0,0 +1,120 @@
+# Procthor Interface - Creating a CEO for testing multible robots in mutliple enviromnents
```

Comment:
Why ceo

Reply:
Its a handler which starts and oversees multiple plans after another. Kind of an dispatcher, couldn't think of a better word/name, so I went with CEO. Do you have any better ideas?